debian/netscanner.manpage

```diff
@@ -0,0 +1,15 @@
+.\" Manpage for netscanner.
+.\" Contact dominic.orsi@gmail.com to correct errors or typos.
+.TH man 1 "22 June 2024" "1.0" "netscanner man page"
+.SH NAME
+netscanner \- view and scan connected networks
+.SH SYNOPSIS
+netscanner
+.SH DESCRIPTION
+Netscanner is a terminal network scanner & diagnostic tool with modern TUI.
+.SH OPTIONS
+Netscanner does not take any options.
+.SH BUGS
+No known bugs.
```

Comment:
Is it possible to put a link here to a github issues ?

Reply:
Yes, updated the manpage, see commit [5a3019](https://github.com/Chleba/netscanner/pull/33/commits/5a301395f3004340d61affffcb9fb3c3f8d3cb66)